java-squid/src/main/java/org/sonar/java/xml/XmlCheck.java

```diff
@@ -17,12 +17,12 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.maven;
+package org.sonar.java.xml;
 
 import org.sonar.plugins.java.api.JavaCheck;
 
 /**
- * Marker interface for the Maven checks.
+ * Marker interface for the Xml checks.
  */
-public interface MavenCheck extends JavaCheck {
+public interface XmlCheck extends JavaCheck {
```

Comment:
This interface does not seem to be useful for anything. I think it can be removed and it will probably simplify your code.